base_layer/common_types/src/transaction.rs

```diff
@@ -38,10 +38,13 @@ pub enum TransactionStatus {
     OneSidedConfirmed = 9,
     /// This transaction is still being queued for initial sending
     Queued = 10,
-    /// This transaction import status is used when a one-sided transaction has been scanned but is unconfirmed
+    /// This transaction import status is used when a coinbase transaction has been scanned but is unconfirmed
     CoinbaseUnconfirmed = 11,
-    /// This transaction import status is used when a one-sided transaction has been scanned and confirmed
+    /// This transaction import status is used when a coinbase transaction has been scanned and confirmed
     CoinbaseConfirmed = 12,
+    /// This transaction import status is used when a coinbase transaction has been scanned but the outputs are not
+    /// currently confirmed on the blockchain via the output manager
+    CoinbaseNotInBlockChain = 13,
```

Comment:
I think we need a unit test to check this status, maybe it could be checked in some of the existing re-org tests.

Reply:
Maybe related to the failing test:
```
FAIL [  20.405s] minotari_wallet::wallet_integration_tests transaction_service_tests::service::test_update_faux_tx_on_oms_validation
```